StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -166,4 +166,10 @@ import Foundation
 
     // MARK: - PaymentSheet Force Success
     case paymentSheetForceSuccess = "mc_force_success"
+
+    // MARK: - PaymentSheet initialization
+    case paymentSheetStartedLoading = "mc_started_loading"
+    case paymentSheetDismissed = "mc_dismiss"
+    case paymentSheetLoaded = "mc_load_success"
+    case paymentSheetLoadFailed = "mc_load_failed"
```

Comment:
Naming nits:
- Can we use the same grammar consistently for these?  e.g. "mc_load_started", "mc_load_succeeded", "mc_load_failed"
- Can we match the case names with the event names?  e.g. Instead of `paymentSheetLoaded = "mc_load_success"`, do `paymentSheetLoadSuccess = "mc_load_success"`